src/webpage/index.html

```diff
@@ -11,71 +11,78 @@
 		<meta content="#4b458c" data-react-helmet="true" name="theme-color">
 		<link href="/style.css" rel="stylesheet">
 		<link href="/themes.css" rel="stylesheet" id="lightcss">
-
+		<style>body,#loading{background:#16191b;}@media(prefers-color-scheme:light){body,#loading{background:#9397bd;}}</style>
```

Comment:
why?

Reply:
This sets the background color to browser preference until the stylesheets load to prevent color flashing

Reply:
That's fair, did you add this to the other pages too?

Reply:
Ooh I didn't, we can do that

Reply:
Yeah, can you go ahead and do that then?

Reply:
So it turned out it doesn't work right on the other pages since they don't have a separate background color from the body. I will think about it and figure something out, sorry it will be later

Reply:
it's ok, no need to say sorry.